samples/tfm_integration/tfm_secure_partition/README.rst

```diff
@@ -0,0 +1,49 @@
+.. _tfm_secure_partition:
+
+TF-M Secure Partition Sample
+############################
+
+Overview
+********
+
+A Secure Partition is an isolated module that resides in TF-M. It exposes a number of functions or "secure services" to other partitions and/or to the non-secure firmware.
+TF-M already contains standard partitions such as crypto, protected_storage, or firmware_update, but it's also possible to create your own partitions.
+
+This sample creates a dummy secure partition and secure service for TF-M and instructs the TF-M build system to build it into the secure firmware.
+The dummy secure service is then called in the main file (in the non-secure firmware).
+
+Everything in the "partition" directory is built by the TF-M build system, refer to :ref:`tfm_build_system` for more details.
+
+This dummy partition has a single secure service, which can index one of 5 dummy secrets inside the partition, and retrieve a hash of the secret.
+
+When adapting this partition for your own purposes, please change all occurences of "dummy_partition", "DUMMY_PARTITION", "dp", and "DP" to your own partition name. Also, look through the CMakeLists.txt both for the secure and non-secure and make relevant changes, as well as the yaml files inside "partition".
```

Comment:
look for the secure what?

Reply:
"Also, look through the CMakeLists.txt both for the secure and non-secure and make relevant changes"
Would be better to invert this to make it easier to read.
"Also, look through both the secure and non-secure CMakeLists.txt file and make relevant changes"